liqd_product/assets/scss/components/_alerts.scss

```diff
@@ -48,9 +48,9 @@ $messages-margin-bottom: 25px;
 
 // If this directly follows the dialog box (see header.scss), add 16px
 // additional padding to the default value (see above).
-.messages:first-child > :first-child {
-    padding-top: calc(16px + #{$padding * 1.2});
-}
+// .messages:first-child > :first-child {
+//     padding-top: calc(16px + #{$padding * 1.2});
+// }
```

Comment:
I only commented this out because I thought we might find something in testing that means we have to adapt it as couldn't find instance of it following dialog box in initial test

Reply:
The dialog box was from mB, the spikey speech bubble line in the header. So, we shouldn't need it here!